### DISCUSSION TITLE
"Getting Started" page and FAQs

### DISCUSSION THREAD

**Discussion by PhilPNguy** [uptane/uptane.github.io#156] [Ideas]
To me, it doesn't make sense to have a "FAQ" in the Getting Started page when the entire thing basically has questions and is a FAQ by itself.

I suggest we just turn the entire page into "FAQ" and change each major heading to be in question format.
E.g.
1. "Key Features of Uptane" becomes "What are the key features of Uptane?"
2. "Getting Started with Implementation" becomes "How do I start implementing Uptane"
3. "Resources" becomes "Where can I find additional Uptane resources?"

Then we move the questions in the current FAQ section to be part of the larger header.

**Comment by jhdalek55** [comment]
I like this suggestion.
